packages/eslint-plugin/src/rules/parameter-properties.ts

```diff
@@ -59,7 +59,6 @@ export default util.createRule<Options, MessageIds>({
             items: {
               $ref: '#/items/0/$defs/modifier',
             },
-            minItems: 1,
```

Comment:
question: why remove this? isn't it better if we keep this for strictness to prevent empty arrays?